fuzzing/config/config.go

```diff
@@ -112,11 +109,53 @@ type FuzzingConfig struct {
 	TestChainConfig config.TestChainConfig `json:"chainConfig"`
 }
 
-// fuzzingConfigMarshaling is a structure that overrides field types during JSON marshaling. It allows FuzzingConfig to
-// have its custom marshaling methods auto-generated and will handle type conversions for serialization purposes.
-// For example, this enables serialization of big.Int but specifying a different field type to control serialization.
-type fuzzingConfigMarshaling struct {
-	TargetContractsBalances []*hexutil.Big
+type ContractBalance struct {
+	big.Int
+}
+
+// UnmarshalJSON parses JSON data into big.Int from empty strings, hex ("0x"),
+// scientific notation (e/E), and base-10 formats
+
+func (cb *ContractBalance) UnmarshalJSON(data []byte) error {
+	s := strings.Trim(string(data), "\"")
```

Comment:
If you remove the quotation marks from the unmarshal, you can remove the trimming here.